lib/src/file_query.dart

```diff
@@ -78,7 +84,9 @@ class _SingleFileQuery implements FileQuery {
   _SingleFileQuery(this.filePath);
 
   @override
-  Iterable<String> generateFilePaths() sync* {
+  Iterable<String> generateFilePaths(
+      {List<String> includePaths = const [],
+      List<String> excludePaths = const []}) sync* {
```

Comment:
What is `sync*`?

Reply:
It indicates that this function is a [synchronous generator](http://dart.goodev.org/articles/language/beyond-async)

Reply:
It makes this method a synchronous generator: https://dart.dev/guides/language/language-tour#generators